tests/drivers/sensor/generic/src/main.c

```diff
@@ -13,6 +13,8 @@
 
 #include <ztest.h>
 #include "dummy_sensor.h"
+#include "toolchain/gcc.h"
```

Comment:
This and the next include shouldn't be needed?

Reply:
Whoops. ClangD auto-inserted and missed it. Thanks!